internal/core/data/v2/controller/http/event.go

```diff
@@ -215,6 +215,32 @@ func (ec *EventController) EventCountByDevice(w http.ResponseWriter, r *http.Req
 	pkg.Encode(eventResponse, w, lc) // encode and send out the response
 }
 
+func (ec *EventController) DeletePushedEvents(w http.ResponseWriter, r *http.Request) {
+	// retrieve all the service injections from bootstrap
+	lc := container.LoggingClientFrom(ec.dic.Get)
+
+	ctx := r.Context()
+	correlationId := correlation.FromContext(ctx)
+
+	var response interface{}
+	var statusCode int
+
+	// Delete all events
```

Comment:
Remove comment. Old `delete all` and now function name is clear what it does.

Reply:
removed